src/main/java/com/mgmtp/radio/support/validator/user/CreateFavoriteSongValidator.java

```diff
@@ -38,17 +41,17 @@ public void validate(Object target, Errors errors) {
 
 	private void validateExists(FavoriteSongDTO favoriteSongDTO, Errors errors) {
 		if (!isUserExisted(favoriteSongDTO.getUserId())) {
-			errors.rejectValue("userId", "", messageSource.getMessage("user.favorite.error.exist.userId", new String[]{}, Locale.getDefault()));
+			errors.rejectValue("userId", "", messageSource.getMessage("validation.error.exist", new String[]{"User Id"}, Locale.getDefault()));
 		}
 
 		if (!isSongExisted(favoriteSongDTO.getSongId())) {
-			errors.rejectValue("songId", "", messageSource.getMessage("user.favorite.error.exist.songId", new String[]{}, Locale.getDefault()));
+			errors.rejectValue("songId", "", messageSource.getMessage("validation.error.exist", new String[]{"Song Id"}, Locale.getDefault()));
 		}
 	}
 
 	private void validateUnique(FavoriteSongDTO favoriteSongDTO, Errors errors) {
 		if (isFavoriteSongExisted(favoriteSongDTO.getUserId(), favoriteSongDTO.getSongId())) {
-			errors.rejectValue("id", "", messageSource.getMessage("user.favorite.error.exist.song", new String[]{}, Locale.getDefault()));
+			errors.rejectValue("id", "", messageSource.getMessage("validation.error.unique", new String[]{"Favorite Song"}, Locale.getDefault()));
```

Comment:
If user or song not exist, do we need to check this?

Reply:
I think we need, because If  both of user and song exist and they pass the validation, then you can not add 1 more duplicate record.